tests/tests/poa.rs

```diff
@@ -110,7 +110,7 @@ mod p2p {
     // after the first_producer stops, second_producer should start producing blocks
     #[tokio::test(flavor = "multi_thread")]
     async fn test_poa_multiple_producers() {
-        const SYNC_TIMEOUT: u64 = 30;
+        const SYNC_TIMEOUT: u64 = 120;
```

Comment:
Why do we need to do this? The test should pass in 10 seconds, we set 30 just because of CI. Why 120?

Reply:
It's too slow in branch coverage mode. I'll just disable it.

Reply:
1c6f607e79dd0c8dbbee8c8610bb949c0bd45b6b